plugins/kubernetes/app/views/kubernetes/deploy_group_roles/index.html.erb

```diff
@@ -41,6 +44,9 @@
           <td><%= deploy_group_role.requests_cpu %> to <%= deploy_group_role.limits_cpu %></td>
           <td><%= deploy_group_role.requests_memory %> to <%= deploy_group_role.limits_memory %></td>
           <td><%= kubernetes_deploy_group_role_replica deploy_group_role.kubernetes_role, deploy_group_role %></td>
+          <% if show_istio_sidecar_ui? %>
+            <td><%= deploy_group_role.inject_istio_annotation? ? 'Yes' : '-' %></td>
```

Comment:
`'Yes' : 'No'` like `app/views/environments/index.html.erb` ... or change both 🤷‍♂

Reply:
I originally had "No" here, but I liked the way it looked better with the dash. It made the text jump out more for the roles that had it enabled when visually scanning the table.